packages/eventual-send/src/index.d.ts

```diff
@@ -55,28 +70,6 @@ export type FilteredKeys<T, U> = {
   [P in keyof T]: T[P] extends U ? P : never;
 }[keyof T];
 
-/**
- * `DataOnly<T>` means to return a record type `T2` consisting only of properties that are *not* functions.
- */
-export type DataOnly<T> = Omit<T, FilteredKeys<T, Callable>>;
-
-// Nominal type to carry the local and remote interfaces of a Remotable.
-export declare class RemotableBrand<L, R> {
-  // The local properties of the object.
-  private localProperties: L;
-
-  // The type of all the remotely-callable functions.
-  private remoteCallable: R;
-}
-
-/**
- * Creates a type that accepts both near and marshalled references that were
- * returned from `Remotable` or `Far`, and also promises for such references.
- */
-export type FarRef<Primary, Local = DataOnly<Primary>> = ERef<
-  Local & RemotableBrand<Local, Primary>
->;
-
 /**
  * `PickCallable<T>` means to return a single root callable or a record type
  * consisting only of properties that are functions.
```

Comment:
More internal-only types:
- `PickCallable`
- `RemoteFunctions`
- `LocalRecord`

The following should be reexported from `handled-promise.js`
- ResolveWithPresenceOptionsBag
- HandledExecutor
- HandledPromiseConstructor
- HandledPromiseStaticMethods - internal to `handled-promise.js`
- Settler

`RemoteKit` is a bit trickier.  I'd like it defined in `E.js` as `EPromiseKit`, reexport it in `index.d.ts`, and change its only existing reference (in `@endo/captp/src/captp.js`)

Reply:
`RemoteFunctions` and `LocalRecord` are only referenced in `E.js` while `PickCallable` is only referenced by `RemoteFunctions`. 

@michaelfig They seem to belong inside `E.js` however keeping them in `types.d.ts` would be needed if we want to preserve the line comments, thoughts?

Reply:
I think moving them to `E.js` is fine.  The comments are useful, but not essential for users of the types.